_layouts/docwithnav.html

```diff
@@ -65,49 +78,61 @@
 		    });
 		}
 		loadonstatechange();
-		/* Load only the doccontent div on click, push state so back/forward browser buttons work, and then animate nav menu */
+		/* Handle clicks in nav menu, content body, and header. Also track history by pushing state (so back/forward browser buttons work)*/
 	    $(document).on("click", 'a', function(e) {
 	        e.preventDefault();
 			var clickedon = $(this),
 				href = $(this).attr('href'),
 				isAnchor = false;
-			if (href.indexOf('#') == 0){//if its an anchor
+			if (href.indexOf('#') == 0){//check if its an in-page anchor
 				isAnchor = true;
-			} else if(href != "#") {//get the absolute url
+			} else if(href != "#") {//otherwise get the absolute url
 				href = this.href;
 			}
-			if(this.host != location.host && href != "javascript:;") {//open external links in new tab/window
-				window.open(href, '_blank');
-			} else if(href != "#" && !clickedon.parents('header').is('#nav') && !clickedon.parents('div').is('#mobile-nav-container')) {//skip empty nav menu containers and any clicks in the banner and banner menu
-				history.pushState(null, null, href);
-				if(isAnchor){//if an in-page anchors
-					var offset = 0;
-					if ($(href).length) {//make sure its a valid anchor, otherwise scroll no where (offset = 0)
-						offset = $(href).offset().top - 155;
-					}
-					$('html, body').animate({scrollTop: offset}, 0);//scroll to anchor
-					return false;
-				} else { //load only the new content into the page (do not load full page)
-					$('#doccontent').load(href + ' #doccontent > *');
-					$('html, body').animate({scrollTop: 0}, 0);
-					if (clickedon.hasClass('gentoc')) {//generate the in-page TOC (load the full page load)
-						window.location = href;
+			if(href == "javascript:;") {//Handle javascript only clicks
+				if ($(this).hasClass('navbuttonarrow')) { //toggle display of the nav menu (hidden by default on small res. displays)
+					$('#wrapper').toggleClass('hidden-sm hidden-xs col-sm-3 col-xs-5');
+					$('.bodywrapper').toggleClass('col-sm-9 col-sm-12 col-xs-7 col-xs-12 toggleposition');
+					$(this).find('img').toggle();
+					if(!($('#wrapper').hasClass('hidden-sm'))) {
+						syncmenutourl();
 					}
-					$.get(href, function(html){//update the page's <title> tags with title from loaded page (so browser shows current title)
-					    var matches = html.match(/<title>([^<]*)/);
-					    if (matches.length>0) {
-					        var title = matches[1];
-					        document.title = title;
-					    }
-					});
 				}
-				if(clickedon.parents().hasClass('bodywrapper')){//animate navigation menu
-					animatenav(findcurnavitem());//is an in-page link
-				} else {
-					animatenav(clickedon);//is a nav menu link
+	            return false;
```

Comment:
Looks right in the file but was a mix of spaces and tabs so ive changed them all to tabs to match how the } above it is indented (maybe that makes a difference in how it displays here in the compare view).